packages/roosterjs-editor-plugins/test/HyperLink/HyperLinkTest.ts

```diff
@@ -0,0 +1,81 @@
+import { IEditor, PluginEventType } from 'roosterjs-editor-types';
+import { Browser } from 'roosterjs-editor-dom';
+import { HyperLink } from '../../lib/HyperLink';
+import { initEditor } from '../TestHelper';
+
+const HREF = 'https://microsoft.github.io/roosterjs';
+const editorId = 'editorId';
+
+describe('HyperLink plugin', () => {
+    let editor: IEditor;
+    let plugin: HyperLink;
+    let anchor: HTMLAnchorElement;
+    let div: HTMLDivElement;
+
+    beforeEach(() => {
+        anchor = document.createElement('a');
+        anchor.href = HREF;
+        plugin = new HyperLink();
+        editor = initEditor(editorId, [plugin]);
+        const editorDiv = (<any>editor).core.contentDiv;
+        if (!editorDiv) {
+            throw 'Unable to find editor div';
+        }
+        div = editorDiv;
+    });
+
+    afterEach(() => {
+        editor.dispose();
+        div.remove();
+    });
+
+    it('Removes title attribute if nothing is being hovered', () => {
+        div.dispatchEvent(new MouseEvent('mouseover'));
```

Comment:
instead of really dispatch an event, we prefer to mock a PluginEvent and call plugin.onPluginEvent() directly since we don't really need to test browser's code, but just test our plugin code only.

Reply:
Same for the others

Reply:
this is handled by mouseover and mouseout events, not `onPluginEvent` 

Should I `(<any>plugin).onMouse` and just test them directly?

Reply:
@ianeli1 I think you can mock/spy editor.addDomEventHandler, so that you can easily get the onMouse callback in an elegant way.